src/cljs/athens/effects.cljs

```diff
@@ -303,10 +303,13 @@
       (when-let [active-el (.-activeElement js/document)]
         (.blur active-el))
       (js/setTimeout (fn []
-                       (let [html-id (str "#editable-uid-" uid)
+                       (let [[uid embed-id]  (db/uid-and-embed-id uid)
+                             html-id         (str "editable-uid-" uid)
                              ;;targets (js/document.querySelectorAll html-id)
                              ;;n       (count (array-seq targets))
-                             el      (js/document.querySelector html-id)]
+                             el              (cond
+                                               embed-id (js/document.querySelector (str "textarea[id^=" html-id "-embed-]"))
```

Comment:
Wow, I actually didn't know this was possible. So what exactly is happening here?

Reply:
updated it to new and added comments 
querySelector can have a regex(ish -- very limited) search https://stackoverflow.com/questions/8714090/queryselector-wildcard-element-match